package.json

```diff
@@ -25,6 +26,8 @@
     "supertest": "^6.3.0"
   },
   "dependencies": {
-    "express": "^4.18.2"
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "uuid": "^9.0.0"
```

Comment:
using `uuid` instead of numeric ids leads to 2 tests failing. But I think it's better than numeric ids ~and I'm lazy to change it~ so, I decided to keep it as it is